sklearn/feature_selection/_rfe.py

```diff
@@ -56,9 +56,14 @@ class RFE(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
         information about feature importance either through a ``coef_``
         attribute or through a ``feature_importances_`` attribute.
 
-    n_features_to_select : int or None (default=None)
-        The number of features to select. If `None`, half of the features
-        are selected.
+    n_features_to_select : int, float or None (default=None)
+        The number of features to select.
+        - If int: the integer is taken as the number of features to select.
+        - If `None`, half of the features are selected.
+        - If float between 0.0 and 1.0: it is taken as the percentage
+        of the available features. For example if you want to select
+        25% of the available features, you would specify 0.25.
```

Comment:
The above 2 lines should be indented at the same level as the "If" above.

Also maybe shorter description,
```
For example, to select 25% of the available features, specify 0.25.
```